infra/main.parameters.json

```diff
@@ -34,6 +34,12 @@
     },
     "storageResourceGroupName": {
       "value": "${AZURE_STORAGE_RESOURCE_GROUP}"
+    },
+    "gptDeploymentName": {
+      "value": "${AZURE_OPENAI_GPT_DEPLOYMENT}"
+    },
+    "chatGptDeploymentName": {
+      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT}"
```

Comment:
Could you also add params for gptModelName chatGptModelName

Also, add all 4 of them to the outputs of main.bicep (at the bottom).  Output var name must match the value you use in main.parameters.json